app/src/main/java/de/dbauer/expensetracker/ui/RecurringExpenseOverview.kt

```diff
@@ -237,6 +332,7 @@ private fun RecurringExpenseOverviewPreview() {
                         ),
                     ),
                 onItemClicked = {},
+                isGridMode = false,
```

Comment:
Implementing previews to streamline testing and debugging is a great idea. If it can be done quickly for this PR I'm all for it. But if not, do you think we should maybe do it as part of another pull request? Maybe just get this one merged and running. Just trying to modularize ours PRs as best I can...

Reply:
I tried it out. It worked but the bottom of my screen was blocked out in white with a next button that didn't work.  I'm getting by with just manually adding test items. Think I'll stick with that for now. But we should definetly implement something like this in the near future.